packages/message-manager/src/types.ts

```diff
@@ -19,3 +19,9 @@ export type PersonalMessageParams = {
 export type TypedMessageParams = {
   data: Record<string, unknown>[] | string | SignTypedDataMessageV3V4;
 } & AbstractMessageParams;
+
+export type Eip7702AuthorizationMessageParams = {
```

Comment:
We'd ideally remove the `message-manager` package entirely except for some lingering references in the extension. We already removed all usages from the `SignatureController` for example.

Could we instead define a simple minimal type in the `KeyringController` itself or the keyring API to remove this coupling?

Reply:
I'm not incredibly familiar with the usage of these types. I see some references within the KeyringController, and within extension.

I've duplicated these types into `KeyringController/types.ts`, but left the previous types in place in the message-manager (without the new 7702 type), with an updated comment (so that existing references will not need to be updated immediately).

Reply:
Sorry for the confusion, I wasn't suggesting we remove the existing references in this PR, but just not continue the pattern for the `Eip7702AuthorizationMessageParams`.

I'll defer to the accounts team if they are happy with the additional refactor of the old types or would prefer an alternate PR.